pkg/worker/k8spodtemplating.go

```diff
@@ -112,6 +126,40 @@ func sanitizePodName(name string) string {
 	return name
 }
 
+func getOwnerReferences() []metav1.OwnerReference {
+	var ownerEnabled bool
+	var ownerName string
+	var ownerUID string
```

Comment:
nit: The `owner` prefix is a bit repepetive, and it's clear enough from the function's overall context.

Suggest just renaming them to `enabled`, `name`, and `uid`.

Also, could be nice if you compacted them a little, like so:

```go
var (
	enabled   bool
	name, uid string
)
```

It's 4 lines instead of 3, but there much less repetition and clear that it's a group of declarations.